tests/integration/helpers.bash

```diff
@@ -19,6 +19,9 @@ HELLO_BUNDLE="$BATS_TMPDIR/hello-world"
 # CRIU PATH
 CRIU="/usr/local/sbin/criu"
 
+# CONFIG_MEMCG_KMEM support
+KMEM="/sys/fs/cgroup/memory/memory.kmem.limit_in_bytes"
```

Comment:
This is not always right, we should parse it from `/proc/self/mountinfo` like we did in `cgroups.bats`.